### PR TITLE
Migration to bem-xjst 8

### PR DESCRIPTION
**Pull request by d3m1d0v**

### CODE REVIEW

File: common.blocks/google-tag-manager/google-tag-manager.bemhtml.js [hunk 1]
```diff
@@ -37,7 +37,9 @@ block('google-tag-manager').replace()(function() {
         },
         {
             tag : 'script',
-            content : scriptContent
+            content : {
+                html : scriptContent
```
Comment (remnev) [inline common.blocks/google-tag-manager/google-tag-manager.bemhtml.js]: why is it necessary?

Comment (tadatuta) [reply common.blocks/google-tag-manager/google-tag-manager.bemhtml.js]: to support escaping in bem-xjst

**Comment by tadatuta** [comment]
@remnev are you ok with the PR?

**Comment by remnev** [comment]
How about the rest of blocks? I mean google-analytics, yandex-metrica ?

File: .enb/make.js [hunk 2]
```diff
@@ -26,7 +26,12 @@ module.exports = function(config) {
                         }
                     },
                     bemhtml : {
-                        tech : 'enb-bemxjst/techs/bemhtml'
+                        tech : 'enb-bemxjst/techs/bemhtml',
+                        options : {
+                            engineOptions : {
+                                xhtml : true
```
Comment (remnev) [inline .enb/make.js]: is it necessary? I heard that it is default value

Comment (d3m1d0v) [reply .enb/make.js]: [Already no](https://github.com/bem/bem-xjst/blob/v8.9.3/lib/bemhtml/index.js#L9)

**Comment by remnev** [comment]
Thank you!

Is it going to be a major fix? Right?

**Comment by tadatuta** [comment]
> Is it going to be a major fix?

it's backward compatible for bem-xjst 7+ so most likely it should work fine for all the users. but if there are some with bem-xjst < 7, they have to update. 

it's up to you to decide if it's a major update.

**Pull request merged**